command/pr_create.go

```diff
@@ -95,7 +95,9 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 
 	// otherwise, determine the head repository with info obtained from the API
 	if headRepo == nil {
-		headRepo, _ = repoContext.HeadRepo()
```

Comment:
This previously assigned `(*api.Repository)(nil)` (a null pointer) to a variable with an interface type, which would make the variable erroneously pass the `head != nil` check